Packs/PrismaCloud/ReleaseNotes/4_2_9.md

```diff
@@ -0,0 +1,9 @@
+
+#### Playbooks
+
+##### Prisma Cloud Remediation - AWS EC2 Security Group Misconfiguration
+
+Removed sections related to Prisma Cloud's deprecated policies.
+##### Prisma Cloud Remediation - AWS EC2 Instance Misconfiguration v2
+
+Removed Prisma Cloud's deprecated policy IDs from task #17. 
```

Comment:
Is this a link?  Why is it underlined?

Reply:
It's not a link - probably a github parsing thing.